sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/storage/StorageBlobHealthIndicator.java

```diff
@@ -35,18 +36,24 @@ public StorageBlobHealthIndicator(BlobServiceAsyncClient blobServiceAsyncClient)
     protected void doHealthCheck(Health.Builder builder) throws Exception {
         if (blobServiceAsyncClient == null) {
             builder.status(NOT_CONFIGURED_STATUS);
-        } else {
-            builder.withDetail(URL_FIELD, blobServiceAsyncClient.getAccountUrl());
-            final Response<BlobServiceProperties> info = blobServiceAsyncClient.getPropertiesWithResponse()
-                                                                               .block(timeout);
-            if (info != null) {
-                builder.up();
-            }
+            return;
         }
+
+        BlobContainerAsyncClient containerAsyncClient = blobServiceAsyncClient.getBlobContainerAsyncClient(
+            NOT_EXISTING_CONTAINER);
+        Response<Boolean> exists = containerAsyncClient.existsWithResponse().block(timeout);
+        assert exists != null;
```

Comment:
Please don't use `assert` in the source code.

Reply:
by default the jvm assertion is turned off

Reply:
Yes, this is to be changed to the following instead:
```java
Objects.requireNonNull(exists, "Error occurred checking the container existence!");

```